packages/gatsby-theme-docs/src/components/markdown.js

```diff
@@ -221,14 +222,15 @@ const CodeBlock = props => {
     props.children.props && props.children.props.children
       ? props.children.props.children
       : props.children;
-  const { splitLanguage, highlightLines } = codeBlockParseOptions(language);
+  const { highlightLines, outputLines } = codeBlockParseOptions(
+    languageToken + props.children.props.metastring
```

Comment:
This is what we get as props

<img width="363" alt="image" src="https://user-images.githubusercontent.com/1110551/68648567-f70ae480-0520-11ea-957e-a21222039c48.png">

Reply:
This changed, not relevant anymore